src/components/DialogContentContainer/__tests__/dialogContentContainer.jest.tsx

```diff
@@ -6,3 +6,8 @@ it("renders correctly with empty props", () => {
   const tree = renderer.create(<DialogContentContainer />).toJSON();
   expect(tree).toMatchSnapshot();
 });
+
+it("renders correctly with data-testid", () => {
```

Comment:
let's add a backlog task to automate this test, kind of a sanity test for all components